code/cmdline/cmdline.cpp

```diff
@@ -1997,8 +1998,8 @@ bool SetCmdlineParams()
 		Cmdline_no_fbo = 1;
 	}
 
-	if ( emissive_arg.found() ) {
-		Cmdline_emissive = 1;
+	if ( emissive_arg.found() && !emissive_power_arg.found()){
+		Cmdline_emissive_power = 0.30f;
```

Comment:
Every time this condition can be true, it will have already set the value to 0.3 in Line 1848, so this if can just be deleted.